src/charm.py

```diff
@@ -32,6 +33,10 @@ class MimirCoordinatorK8SOperatorCharm(CharmBase):
 
     def __init__(self, *args):
         super().__init__(*args)
+
+        self._nginx_container = self.unit.get_container("nginx")
+        self._nginx_config_path = "/etc/nginx/nginx.conf"
```

Comment:
Do we need this property since we have it in nginx.py?

Reply:
Good eye, I forgot that there; I'll remove it!